app/src/main/res/layout-land/profile_edit_activity.xml

```diff
@@ -0,0 +1,235 @@
+                android:layout_height="wrap_content"
+                android:fontFamily="sans-serif"
+                android:text="@string/profile_edit_allow_download_heading"
+                android:textColor="@color/oppiaPrimaryText"
+                android:textSize="16dp" />
+
+              <TextView
+                android:id="@+id/profile_edit_allow_download_sub"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:fontFamily="sans-serif"
+                android:text="@string/profile_edit_allow_download_sub"
+                android:textSize="14sp" />
+            </LinearLayout>
+
+            <Switch
+              android:id="@+id/profile_edit_allow_download_switch"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:layout_marginTop="4dp"
+              android:layout_marginEnd="24dp"
+              app:layout_constraintEnd_toEndOf="parent"
+              app:layout_constraintTop_toTopOf="parent" />
+          </androidx.constraintlayout.widget.ConstraintLayout>
+
+          <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="@color/grey" />
+        </LinearLayout>
+      </LinearLayout>
+
+      <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="24dp"
+        android:layout_marginBottom="8dp"
+        android:clipToPadding="false"
+        android:orientation="vertical"
+        android:visibility="@{viewModel.profile.isAdmin ? View.GONE : View.VISIBLE}"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/linearLayout"
+        app:layout_constraintVertical_bias="1.0">
+
+        <View
+          android:layout_width="match_parent"
+          android:layout_height="1dp"
+          android:background="@color/grey" />
+        <Button
+          android:id="@+id/profile_delete_button"
+          style="?android:attr/borderlessButtonStyle"
+          android:layout_width="match_parent"
+          android:layout_height="48dp"
+          android:background="@color/white"
+          android:fontFamily="sans-serif"
+          android:gravity="start|center_vertical"
+          android:paddingStart="36dp"
+          android:text="@string/profile_edit_delete"
+          android:textAllCaps="false"
+          android:textColor="@color/red"
+          android:textSize="16sp" />
+
+
+        <View
+          android:layout_width="match_parent"
+          android:layout_height="1dp"
+          android:background="@color/grey" />
+      </LinearLayout>
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+  </ScrollView>
+</layout>
```

Comment:
Instead of following all changes suggesterd in this file, you can simply press `Control + Alt + L` - reformatting shortcut to fix the formatting.